app/views/authors/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Author Show Page</h1>
+
+<section id="author-info">
+  <li><%= @author.name %><li>
+</section>
+
+<section id="authors-books">
+    <% @books.each do |book| %>
+  <div id=<%= "book-#{book.id}" %>>
```

Comment:
Looks like the user story also specifies that there should be a "small image of the cover" in the author show page